Building: jar vulnerabilities for jackson-core

The master pom.xml calls for elasticsearch 2.3.1.  mvn clean install does not like jackson-core 2.6x. Switched to elasticsearch 2.4.6 and now it doesn't like jackson-core 2.8x.

Dependency-Check Failure:
[ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater then '4.0':
[ERROR] jackson-core-2.8.1.jar: CVE-2016-7051
Mentioned here:
https://groups.google.com/forum/#!topic/jackson-user/esVUkLkkD8Y
Found suppression here:
https://github.com/FasterXML/jackson-core/issues/371

--- a/dependency-suppression.xml
+++ b/dependency-suppression.xml
@@ -5,4 +5,12 @@
         <cve>CVE-2015-5262</cve>
         <cve>CVE-2014-3577</cve>
     </suppress>
+    <suppress>
+        <!-- false positive only affects jackson-dataformat-xml not jackson-core-->
+        <gav regex="true">^com\.fasterxml\.jackson\.(core:jackson-core|dataformat:jackson-dataformat-cbor):.*$</gav>
+        <!-- see https://github.com/FasterXML/jackson-dataformat-xml/issues/190 -->
+        <cve>CVE-2016-7051</cve>
+        <!-- see https://github.com/jeremylong/DependencyCheck/issues/517 -->
+        <cve>CVE-2016-3720</cve>
+    </suppress>
 </suppressions>

